server/tests/fixtures/orm_session.py

```diff
@@ -1,18 +1,21 @@
+import os
 import pytest
 import pytest_asyncio
 import sqlalchemy
 import sqlalchemy.orm
-from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
 
 from npg.porchdb.models import Base
+from npg.porchdb.db import session_factory, deploy_schema, close_engine
 
 
 @pytest.fixture
 def sync_session():
-    'A synchronous sqlalchemy scoped session, connected to sqlite'
+    '''
+    A synchronous sqlalchemy scoped session, connected to sqlite
+    Use for tests that don't need a running npg_porch server
+    '''
 
     sqlite_url = 'sqlite+pysqlite:///:memory:'
```

Comment:
Could we use the value from the `config` disc in `npg.porchdb.db`. Probably have to create `get_config` accessor there?

Reply:
We might not need a synch connection at he end, let's not put more work into it